include/cglm/structs/mat4.h

```diff
@@ -47,16 +307,154 @@ glms_mat4_mulv(mat4 m, vec4s v) {
  * @param[in]  m    mat4(affine transform)
  * @param[in]  v    vec3
  * @param[in]  last 4th item to make it vec4
- * returns          result vector (vec3)
+ * @returns         result vector (vec3)
  */
 CGLM_INLINE
 vec3s
-glms_mat4_mulv3(mat4 m, vec3s v, float last) {
+glms_mat4_mulv3(mat4s m, vec3s v, float last) {
   vec3s r;
-  glm_mat4_mulv3(m, v.raw, last, r.raw);
+  glm_mat4_mulv3(m.raw, v.raw, last, r.raw);
   return r;
 }
 
+/*!
+ * @brief transpose mat4 and store in dest
+ *
+ * source matrix will not be transposed unless dest is m
+ *
+ * @param[in]  m    matrix
+ * @returns         result
+ */
+CGLM_INLINE
+mat4s
+glms_mat4_transpose_to(mat4s m) {
+  mat4s r;
```

Comment:
let's remove `_transpose_to` version for now.

Reply:
Will do.